### ISSUE TITLE
Not able to play audio files in Android build

### ISSUE THREAD

**Issue by Javasamurai** [mini2Dx/mini2Dx#276]
Exception while running

`Gdx.audio.newSound(Gdx.files.internal("sfx/point1.wav"))`
```
java.lang.NullPointerException: Attempt to invoke virtual method 'void com.badlogic.gdx.utils.LongArray.add(long)' on a null object reference
        at com.badlogic.gdx.backends.android.AndroidMini2DxAudio.appendRecentSoundId(AndroidMini2DxAudio.java:96)
        at com.badlogic.gdx.backends.android.AndroidMini2DxSound.play(AndroidMini2DxSound.java:58)
        at com.badlogic.gdx.backends.android.AndroidMini2DxSound.play(AndroidMini2DxSound.java:48)



```

Tried on ogg, mp3, wav files. Works on the desktop version of the game.

**Comment by tomcashman** [comment]
What version of mini2Dx are you using?

**Comment by Javasamurai** [comment]
`1.8.0` It's a bit old. I have n't updated in a while.

**Comment by tomcashman** [comment]
Looks like this was fixed in `1.9.3`
